WordPress/gutenberg
# Components: refactor`Draggable` to pass `exhaustive-deps`

## What?
Updates the `Draggable` component to satisfy the `exhaustive-deps` eslint rule (specifically by updating the `ControlPoints` subcomponent)

## Why?
Part of the effort in https://github.com/WordPress/gutenberg/pull/41166 to apply `exhuastive-deps` to the Components package

## How?
Add the three values returned as an object by `useMemo` to the dependency array so they're current.

Previously this was an empty array so it would only occur on the initial render. I'm not 100% sure if was intentional or not. My testing of this change didn't raise any red flags, but I'd love some feedback from someone more familiar with React native (cc @geriux)

## Testing Instructions
1. From your local Gutenberg directory, run `npx eslint --rule 'react-hooks/exhaustive-deps: warn' packages/components/src/draggable`
2. Confirm that the linter returns no errors
3. Confirm unit tests still pass
4. Run Storybook locally, confirm the components stories and/or docs still work as expected
